frontend/src/App.tsx

```diff
@@ -1,20 +1,37 @@
-import { StatusBar } from "expo-status-bar";
-import { StyleSheet, Text, View } from "react-native";
+import { DefaultTheme, NavigationContainer } from "@react-navigation/native";
+import { View, StyleSheet } from "react-native";
 
-export default function Sample() {
+import Header from "./components/Header/Header";
+import AppNavigator from "./navigation/AppNavigator";
+import { linking } from "./navigation/navigationLinks";
+
+const App = () => {
   return (
-    <View style={styles.container}>
-      <Text>Open up App.tsx to start working on your app!</Text>
-      <StatusBar style="auto" />
-    </View>
+    <NavigationContainer
+      linking={linking}
+      theme={{
+        ...DefaultTheme,
+        colors: { ...DefaultTheme.colors, background: "white" },
```

Comment:
This is just the way that prettier formatted it. It would look like your format if the number of characters in the line exceeds the character limit of 80.